src/assets/stylesheets/layout/_topics.scss

```diff
@@ -9,18 +9,14 @@
     border: 1px solid #6695FF;
     padding: 1rem;
     border-radius: 0.25rem;
-    flex-basis: 33%;
-    margin: 0 1rem;
+    flex-basis: 32%;
     text-decoration: none;
     color: var(--codacy-primary);
     &:hover, &:focus {
       color: var(--codacy-primary);
       box-shadow: 0px 6px 5px #6695ff26;
     }
 
-    &:first-child { margin-left: 0 }
-    &:last-child { margin-right: 0 }
```

Comment:
The `justify-content: space-between;` ensures that the first and last cards are stuck to the left and right-hand sides of their container, so it's not necessary to specify these margins.